msg/ko_KR.utf8/cubrid.msg

```diff
@@ -1513,6 +1513,11 @@ $set 7 MSGCAT_SET_PARSER_SYNTAX
 127 커맨트의 최대 길이는 %1$d 바이트입니다.
 128 JSON_OBJECTAGG 인자의 갯수가 잘못되었습니다.
 129 PARALLEL 인자 값은 %1$d에서 %2$d까지 허용됩니다.
+130 이름 바꾸기 전과 후의 소유자가 다릅니다.\n  이름 바꾸기는 소유자를 변경할 수 없습니다.
+131 잘못된 형식의 alter synonym 구문입니다.\n  ALTER [PUBLIC|PRIVATE] SYNONYM [<user_name>.]<synonym_name> FOR [<user_name>.]<target_name> [COMMENT 'comment_string']
+132 잘못된 형식의 create synonym 구문입니다.\n  CREATE [OR REPLACE] [PUBLIC|PRIVATE] SYNONYM [<user_name>.]<synonym_name> FOR [<user_name>.]<target_name> [COMMENT 'comment_string']
+133 잘못된 형식의 drop synonym 구문입니다.\n  DROP [PUBLIC|PRIVATE] SYNONYM [IF EXISTS] [<user_name>.]<synonym_name>
+134 잘못된 형식의 rename synonym 구문입니다.\n  RENAME [PUBLIC|PRIVATE] SYNONYM [<user_name>.]<old_synonym_name> {AS|TO} [<user_name>.]<new_synonym_name>
```

Comment:
Referring to the following, between the new line(`\n`) and the next sentence, there is a single blank.
```
98 잘못된 형식의 second 함수입니다.\n SECOND( expression ) 
```
How about doing the same for the consistency of error messages?

Reply:
```suggestion
130 이름 바꾸기 전과 후의 소유자가 다릅니다.\n 이름 바꾸기는 소유자를 변경할 수 없습니다.
131 잘못된 형식의 alter synonym 구문입니다.\n ALTER [PUBLIC|PRIVATE] SYNONYM [<user_name>.]<synonym_name> FOR [<user_name>.]<target_name> [COMMENT 'comment_string']
132 잘못된 형식의 create synonym 구문입니다.\n CREATE [OR REPLACE] [PUBLIC|PRIVATE] SYNONYM [<user_name>.]<synonym_name> FOR [<user_name>.]<target_name> [COMMENT 'comment_string']
133 잘못된 형식의 drop synonym 구문입니다.\n DROP [PUBLIC|PRIVATE] SYNONYM [IF EXISTS] [<user_name>.]<synonym_name>
134 잘못된 형식의 rename synonym 구문입니다.\n RENAME [PUBLIC|PRIVATE] SYNONYM [<user_name>.]<old_synonym_name> {AS|TO} [<user_name>.]<new_synonym_name>
```

Reply:
It could apply for the other message files

Reply:
Other similar error messages are indented by 2 spaces.

Is there any reason to shorten it to 1 space?

Reply:
I found that error number 30's message has 2 spaces. The consistency of the message is already broken. I'm gonna resolve this conversion.

Reply:
All right. Thank you.